apistructs/cluster_dialer.go

```diff
@@ -47,3 +51,21 @@ func (c ClusterDialerClientType) MakeClientKey(clusterKey string) string {
 	}
 	return fmt.Sprintf("%s-client-type-%s", clusterKey, c)
 }
+
+type ClusterDialerDataKey string
```

Comment:
not data key.
ClusterDialerClientDetailKey

Reply:
adjusted